include/transaction/esdt.h

```diff
@@ -4,15 +4,43 @@
 #include "transaction.h"
 #include "scarguments.h"
 
-#define NO_FUNCTION std::string()
-#define NO_PARAMETERS SCArguments()
-
-#define ESDT_GAS_LIMIT_NO_FUNCTION 500000
 #define ESDT_TRANSFER_PREFIX std::string("ESDTTransfer")
+#define ESDT_TRANSFER_GAS_LIMIT_NO_FUNCTION 500000
+#define ESDT_TRANSFER_NO_FUNCTION std::string()
+#define ESDT_TRANSFER_NO_ARGUMENTS SCArguments()
+
+#define ESDT_ISSUANCE_PREFIX std::string("issue")
+#define ESDT_ISSUANCE_VALUE "50000000000000000" //(0.05 EGLD)
```

Comment:
Shouldn't this be 5?

Reply:
it was changed to 0.05 egld

Reply:
Info is from here: https://docs.elrond.com/developers/esdt-tokens/. It seemns to be 0.05 EGLD